export_emails.py

```
@@ -0,0 +1,226 @@
+import urllib.request
+import urllib.parse
+import os
+import argparse
+import string
+from datetime import datetime
+
+# Configurações da API do Grafana do Top Massagens
+GRAFANA_URL = "https://grafana.topmassagens.com.br/api/ds/query?ds_type=mysql"
+DATASOURCE_UID = "dfpqllglhd88wf"
+AUTH_USER = "timetop"
+AUTH_PASS = "77308515"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "AUTH_PASS"

(S105)